src/app/patterns/creation-flow/creation-dialog/creation-dialog.component.html

```diff
@@ -0,0 +1,86 @@
+    <span [routerLink]="['/']" class="cursor-pointer text-upper">Sandbox</span>
+    <span flex></span>
+    <a mat-icon-button matTooltip="Docs" matTooltipPosition="before" matTooltipDelay="500" href="https://teradata.github.io/covalent/" target="_blank">
+      <mat-icon>chrome_reader_mode</mat-icon>
+    </a>
+    <a mat-icon-button matTooltip="Github" matTooltipPosition="before" matTooltipDelay="500" href="https://github.com/Teradata/covalent-resources" target="_blank">
+      <mat-icon svgIcon="assets:github"></mat-icon>
+    </a>
+  </div>
+  <td-layout-manage-list [opened]="media.registerQuery('gt-sm') | async" [mode]="(media.registerQuery('gt-sm') | async) ? 'side' : 'push'"
+    [sidenavWidth]="(media.registerQuery('gt-xs') | async) ? '257px' : '100%'">
+    <mat-toolbar td-sidenav-content>
+      <mat-toolbar-row>
+        <mat-icon class="push-left-sm push-right-sm">add_circle</mat-icon>
+        <span class="push-left">Creation Flow</span>
+      </mat-toolbar-row>
+    </mat-toolbar>
+    <mat-nav-list td-sidenav-content [tdLayoutManageListClose]="!media.query('gt-sm')">
+      <a mat-list-item [routerLink]="['/patterns','creation-flow']">
+        <mat-icon matListIcon>view_list</mat-icon>
+        Management List
+      </a>
+      <a mat-list-item [routerLink]="['/patterns','creation-flow','card-over']">
+        <mat-icon matListIcon>view_day</mat-icon>
+        Card Over
+      </a>
+      <a mat-list-item [class.active]="true" [routerLink]="['/patterns','creation-flow','dialog']">
+        <mat-icon matListIcon>filter_none</mat-icon>
+        Dialog
+      </a>
+      <a mat-list-item [routerLink]="['/patterns','creation-flow','sidesheet']">
+        <mat-icon matListIcon>format_indent_increase</mat-icon>
+        Sidesheet
+      </a>
+    </mat-nav-list>
+    <mat-toolbar>
+      <mat-toolbar-row>
+        <button mat-icon-button tdLayoutManageListOpen [hideWhenOpened]="true">
+          <mat-icon>arrow_back</mat-icon>
+        </button>
+        <div td-toolbar-content layout="row" layout-align="start center" flex>
+          <span>Dialog</span>
+          <span flex></span>
+          <a mat-icon-button [href]="baseURL + '/patterns/creation-flow/creation-dialog/creation-dialog.component.html'" target="_blank"
+            class="push-right">
+            <mat-icon color="primary" matTooltip="Open in Stackblitz" matTooltipPosition="before" matTooltipDelay="500">code</mat-icon>
+          </a>
+          <button mat-raised-button *ngIf="(media.registerQuery('gt-xs') | async)" color="accent" class="text-upper" (click)="openDialog()">create</button>
+        </div>
+      </mat-toolbar-row>
+    </mat-toolbar>
+    <mat-card class="push">
+      <mat-card-title>When to use a dialog</mat-card-title>
+      <mat-divider></mat-divider>
+      <mat-card-content>
+        <td-message label="Click create button to see Card Over creation pattern" color="accent" icon="info" class="push-bottom"></td-message>
```

Comment:
Instead of this message, we can just place this button here.

Reply:
The pattern requires the button in the toolbar